projects/api/prisma/migrations/20231122143609_init/migration.sql

```diff
@@ -0,0 +1,131 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "userId" SERIAL NOT NULL,
```

Comment:
I suggest you check guidGUID
`userId uuid NOT NULL default public.uuid_generate_v4(),
`
however, if you just need a simple, sequentially ordered identifier primarily for your ids records and you do not require global uniqueness, a serial number might be sufficient

Reply:
yes originally I thought of using uuid but we really don't need global uniqueness in the id of our products